### PR TITLE
added container analysis samples

### PR DESCRIPTION
**Pull request by DanSanche**
Added samples for new [Container Analysis](https://cloud.google.com/container-registry/docs/container-analysis) Ruby library. This is my first experience with Ruby, so be thorough

I didn't include the beta library into the repo yet since it's pre-release, so the tests won't work. Please review the code style, and I'll add the library and enable tests before merging

### CODE REVIEW

**Comment by TheRoyalTnetennba** [comment]
Could you fix the errors raised by running `bundle exec rspec`?

**Comment by DanSanche** [comment]
I believe the errors are because I didn't include the library in the PR (it's pre-release, so I'd want to talk with the team before making it public on github). Is it possible to get a code review first, and I'll then add in the library or import later before merging?

**Comment by TheRoyalTnetennba** [comment]
Sorry, I meant `bundle exec rubocop`

**Comment by DanSanche** [comment]
sorry, I wasn't aware of the style checker. Should be better now

File: container-registry/container-analysis/sample.rb [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+    puts "Message #{count}: #{received_message.data}"
+    received_message.acknowledge!
+  end
+  subscriber.start
+  # Wait for incomming occurrences
+  sleep timeout_seconds
+  subscriber.stop.wait!
+  subscription.delete
+  # Print and return the total number of Pub/Sub messages received
+  puts "Total Messges Received: #{count}"
```
Comment (TheRoyalTnetennba) [inline container-registry/container-analysis/sample.rb]:
```suggestion
  puts "Total Messages Received: #{count}"
```

File: container-registry/container-analysis/Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  ruby
+
+DEPENDENCIES
+  google-cloud-devtools-containeranalysis!
+  google-cloud-pubsub (~> 0.35.0)
+  public_suffix (= 3.0.3)
+  rspec
+
+BUNDLED WITH
+   2.0.1
```
Comment (TheRoyalTnetennba) [inline container-registry/container-analysis/Gemfile.lock]: will probably have to be changed to 1.17.3 for CI to pass

**Comment by DanSanche** [comment]
opened new PR here: https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/435

**Pull request closed without merging**